js_modules/dagster-ui/packages/ui-core/src/assets/AssetTableFragment.tsx

```diff
@@ -26,6 +26,10 @@ export const ASSET_TABLE_DEFINITION_FRAGMENT = gql`
     autoMaterializePolicy {
       policyType
     }
+    automationCondition {
```

Comment:
I wonder if the `autoMaterializePolicy` bit above these lines is still in use?

Reply:
updated!